eras/babbage/impl/CHANGELOG.md

```diff
@@ -1,5 +1,9 @@
 # Version history for `cardano-ledger-babbage`
 
+## 1.4.3.1x
```

Comment:
Just noticed a stray `x` at the end of the version numbering here.

Reply:
Thank you. I got rid of it.